### PR TITLE
Migrate to Lita 4.x style

### PR DESCRIPTION
**Pull request by ursm**

### CODE REVIEW

**Comment by fukayatsu** [comment]
寿司ゆき:thanks

File: lib/lita/handlers/irkit.rb [hunk 1]
```diff
@@ -48,30 +47,30 @@ def ir_send_all_off(response)
...
-    private
+      def ir_migrate(response)
+        keys = Lita.redis.keys('irkit:messages:*')
 
-      def key_for_cmd(cmd)
-        "irkit:messages:#{cmd}"
-      end
+        Lita.redis.pipelined do
+          keys.each do |key|
+            Lita.redis.rename key, key.sub(/^irkit:messages:/, 'handlers:irkit:')
```
Comment (fukayatsu) [inline lib/lita/handlers/irkit.rb:59]: I'd like to use `handlers:irkit:messages:` to enable to store `handlers:irkit:schedules:` or something in the future. Maybe I change it after merging this pull-request.

File: lib/lita/handlers/irkit.rb [hunk 2]
```diff
@@ -4,33 +4,32 @@
...
-      end
+      config :deviceid,  required: true, default: ENV['IRKIT_DEVICEID']
+      config :clientkey, required: true, default: ENV['IRKIT_CLIENTKEY']
 
       route /^ir list/,            :ir_list,         command: false, help: { "ir list"                      => "list irkit command names" }
       route /^ir send (.+)/,       :ir_send,         command: false, help: { "ir send [command_name]"       => "send irkit command" }
       route /^ir all off/,         :ir_send_all_off, command: false, help: { "ir all off"                   => "send irkit commands which end with 'off'" }
       route /^ir register (.+)/,   :ir_register,     command: true,  help: { "ir register [command_name]"   => "register irkit command" }
       route /^ir unregister (.+)/, :ir_unregister,   command: true,  help: { "ir unregister [command_name]" => "unregister irkit command" }
+      route /^ir migrate/,         :ir_migrate,      command: true
```
Comment (fukayatsu) [inline lib/lita/handlers/irkit.rb:15]: TODO for me: Add `ir migrate` to https://github.com/fukayatsu/lita-irkit#usage and describe migrating

File: lib/lita/handlers/irkit.rb (hunk 1 above)
Comment (ursm) [inline lib/lita/handlers/irkit.rb:59]: I see. I have an idea about this.
How about adding `#send_command` method to this handler and using that from a scheduler?

``` ruby
cron '* 9 * * *', :light_on
def light_on
  Handlers::Irkit.new(robot).send_command 'light on'
end
```

Comment (ursm) [inline lib/lita/handlers/irkit.rb:59]: Sorry, I misunderstood. I approve your opinion.

**Comment by ursm** [comment]
Thank you! :cat2:

**Pull request merged**